app/submitter/submitter.py

```diff
@@ -141,3 +143,46 @@ def send_message(self, message, tx_id, questionnaire_id, case_id=None):
         finally:
             if connection:
                 self._disconnect(connection)
+
+
+class GCSFeedback:
+    def __init__(self, bucket_name):
+        client = storage.Client()
+        self.bucket = client.get_bucket(bucket_name)
+        self.bucket.retention_period = 86400
+
+    def upload(self, schema, feedback_data, session_data):
+        object_key = str(uuid4())
+        blob = self.bucket.blob(object_key)
+        blob.metadata = {
+            "feedback_count": session_data.feedback_count + 1,
+            "form_type": schema.form_type,
+            "language_code": session_data.language_code,
+            "object_key": object_key,
+            "region_code": schema.region_code,
+            "submitted_at": session_data.submitted_time,
+            "tx_id": session_data.tx_id,
+        }
+        blob.upload_from_string(str(feedback_data).encode("utf8"))
```

Comment:
I think it would be good to set the correct content type for the blob. The `upload_from_string` method also accepts a `content_type` which we should set to `application/json` as it defaults to `text/plain`.